execution/executor/src/metrics.rs

```diff
@@ -19,7 +22,8 @@ pub static APTOS_EXECUTOR_APPLY_CHUNK_SECONDS: Lazy<Histogram> = Lazy::new(|| {
         // metric name
         "aptos_executor_apply_chunk_seconds",
         // metric description
-        "The time spent in seconds of applying txn output chunk in Aptos executor"
+        "The time spent in seconds of applying txn output chunk in Aptos executor",
+        exponential_buckets(/*start=*/ 1e-3, /*factor=*/ 2.0, /*count=*/ 12).unwrap(),
```

Comment:
so this is 1ms to > 4 seconds?

Reply:
Yes. 

BTW some other place in this PR I use 1e-6 as the start, which is 1us to >4 seconds

Reply:
Any downside to extend that to a minute or two? (for those you chose to start from 1ms -- because I do push it to tens of seconds of block execution time etc when i play with the benchmark with large block sizes, for example.)

Reply:
Sure. The overhead should be very minimal.